internal-api/src/main/proto/systems/ajax/motrechko/airguardian/commonresponse/event/emergency_event.proto

```diff
@@ -1,21 +1,18 @@
 syntax = "proto3";
 
 import "google/protobuf/timestamp.proto";
+import "google/type/latlng.proto";
 
 package systems.ajax.motrechko.airguardian.commonresponse.event;
 
 option java_multiple_files = true;
 
-message Coordinates {
-  double latitude = 1;
-  double longitude = 2;
-}
-
 enum EmergencyEventType {
-  ROBBERY = 0;
-  FIRE = 1;
-  SHOOTING = 2;
-  OTHER = 3;
+  UNKNOWN = 0;
```

Comment:
protobuf suggests following naming for unset enum _UNSPECIFIED, so rename please to EMERGENSY_EVENT_TYPE_UNSPECIFIED
and what about other enums?